catroid/build.gradle

```diff
@@ -475,7 +475,7 @@ dependencies {
     natives "com.badlogicgames.gdx:gdx-box2d-platform:$gdxVersion:natives-armeabi-v7a"
     natives "com.badlogicgames.gdx:gdx-box2d-platform:$gdxVersion:natives-arm64-v8a"
 
-    implementation "com.google.android.gms:play-services-auth:$playServicesVersion"
+    implementation "com.google.android.gms:play-services-auth:17.0.0"
```

Comment:
Please use the variable again

Reply:
I cannot, because somehow the stage breaks then (I think). I played around pretty long with these versions and it seems that this one has to be 17.0.0 and the com.google.android.gms:play-services-analytics 17.0.1.